### ISSUE TITLE
Rewrite as a set of GitHub actions

### ISSUE THREAD

**Issue by freaktechnik** [mozillach/gh-projects-content-queue#186]
Not only would this make hosting much simpler, I believe it would make customizing it much more accessible, too. The idea from #77 kind of touches on this idea, but I think actions are better suited for what this tries to do, at least for the processing part, not necessarily for the incoming side. I'd have to look at github actions closer to figure out if they can be the incoming side, too.

**Comment by MichaelKohler** [comment]
As far as I know issue events can be triggers as well. I like this idea :)

**Comment by freaktechnik** [comment]
Yeah, but turning external triggers into issues would still likely require an external service.

**Comment by MichaelKohler** [comment]
I don't have all the external triggers in mind, but the ical could for example be checked on a cron trigger.

**Comment by freaktechnik** [comment]
Ah right, since all the incoming stuff is currently based on polling it could very well be done using a scheduled workflow.
